File: Python/Human.py
# membuat kelas human
class Human:
    # atribut privat kelas Human
    __nik = 0
    __nama = ""
    __jenkel = ""

    # konstruktor dengan parameter
    def __init__(self, nik, nama, jenkel):
        #set nilainya
        self.__nik = nik
        self.__nama = nama
        self.__jenkel = jenkel
    
    # tetapkan nik
    def setNik(self, nik):
        self.__nik = nik
    
    # get nik
    def getNik(self):
        return self.__nik
    
    # tetapkan nama
    def setNama(self, nama):
        self.__nama = nama

    # get nama
    def getNama(self):
        return self.__nama
    
    # tetapkan jenkel
    def setJenkel(self, jenkel):
        self.__jenkel = jenkel
    
    # get jenkel
    def getJenkel(self):
        return self.__jenkel

File: CPP/Human.cpp
// library
#include <string>
#ifndef Human_H
#define Human_H

using namespace std;

// membuat kelas Human
class Human{
    // atribut yang diperlukan
    private:
        string nik;
        
    public:
        string nama;
        string jenkel;
    
    public:
        // konstruktor kosong
        Human() {
        }

        // konstruktor dengan parameter
        Human(string nik, string nama, string jenkel) {
            // set nilainya
            this->nik = nik;
            this->nama = nama;
            this->jenkel = jenkel;
        }

        // tetapkan nama
        void setNama(string nama) {
            this->nama = nama;
        }

        // get nama
        string getNama() {
            return this->nama;
        }

         // tetapkan nik
        void setNik(string nik) {
            this->nik = nik;
        }

        // get nik
        string getNik() {
            return this->nik;
        }

         // tetapkan jenkel
        void setJenkel(string jenkel) {
            this->jenkel = jenkel;
        }

        // get jenkel
        string getJenkel() {
            return this->jenkel;
        }
        
        // deconstruct
        ~Human() {
        }
};

#endif